client/src/ui/base/_themes.scss

```
@@ -210,7 +210,7 @@
   --baseline-limited-check: #1e8e3e;
   --baseline-limited-cross: #ea8600;
 
-  --curriculum-bg-color: #fcefe2;
+  --curriculum-bg-color: #faf0e3;
```

Revert to previous color:

```suggestion
  --curriculum-bg-color: #fcefe2;
```